src/project.c

```diff
@@ -611,6 +618,10 @@ static void show_project_properties(gboolean show_build)
 	gtk_entry_set_text(GTK_ENTRY(e.patterns), entry_text);
 	g_free(entry_text);
 
+	gtk_toggle_button_set_active(
+		GTK_TOGGLE_BUTTON(ui_lookup_widget(e.dialog, "checkbutton_project_dialog_file_relative")),
```

Comment:
Didn't you lookup this widget above? Can't you use that pointer.

Reply:
Right ! I will do that